### DISCUSSION TITLE
Found disagreement in the SM binned differential BR of B2KTauTau with the paper

### DISCUSSION THREAD

**Discussion by rajeevneutrino** [flav-io/flavio#227] [Q&A]
Dear authors,

The SM value obtained for the "Binned differential branching ratio of B±→K± τ+ τ−" in q2=[15,22] from flavio is 1.79e-8. I wonder why this value doesn't match with the value reported in arXiv:1712.01919v1 which is 1.20e-7. In fact, from my own code I get similar value as reported in the paper.

I find similar disagreements for B2K*tautau (q2=15,19), Bs2Phitautau (q2=15,18.8) decays. However, the BR(Bs2tautau) is perfectly fine. ​​

Kindly help me

Thanks and Regards
Rajeev
![Screenshot from 2023-05-19 16-42-03](https://github.com/flav-io/flavio/assets/71543739/1f104802-bb55-4cc8-9f7b-f862e3b1ba85)

**Comment by peterstangl** [comment]
Hi @rajeevneutrino,

the disagreement that you find might be due to different definitions of what "binned branching ratio" means. Note that `<dBR/q2>` in `flavio` is a bin-averaged differential branching ratio, i.e. it corresponds to the integral over the differential branching ratio divided by the bin width. Your reference might use a different definition without dividing by the bin width.

**Reply by rajeevneutrino** [reply]
Dear @peterstangl 

Thank you for clarifying my doubt.

Regards.
